src/behaviours/DragAndDrop/reducer.js

```diff
@@ -95,6 +104,15 @@ const triggerDrop = (state, source) => {
     },
   });
 
+  filter(hits.targets, { willAccept: true })
+    .forEach((target) => {
+      target.onDragEnd(hits.source);
+    });
+
+  if (some(hits.obstacles, { isOver: true, willAccept: true })) {
```

Comment:
Is `willAccept` needed here? It seems like if a node is over an obstacle, then this should return.

If that check is removed, then can we also remove the `withHandlers({ accepts: () => () => true  })` composed props for the obstacles?

Reply:
Interesting question. I was definitely modeling after DropTargets here; I thought we could potentially have a case where a DropObstacle doesn't matter in a certain situation. But we could always add an acceptance check back in if that arbitrary situation ever arises. All the obstacles block all drop actions right now anyway.